### PR TITLE
bagelstation fixes

### LINKED ISSUE #15107: BagelStation, no way to activate shower without using Alt-click
(WEB REPORT BY: wulfel REMOTE: 172.93.109.202:7777)
Because the shower curtain is on top of the shower itself, you can't actually click the shower to turn it on unless you Alt-click to show everything in the tile in your tab window.

### LINKED ISSUE #15920: Mining conveyers on bagel are all wrong and don't work right
(WEB REPORT BY: lavoss REMOTE: 172.93.109.202:7777)
Try using the conveyers and you'll see exactly where the multiple issues are, biggest issue is that supplies dont get vended out, but theres animation issues too. Use its from the space access starting point.

### LINKED ISSUE #16385: Bagel Station Kick Mine
Was pointed out in #16292, but not removed before merge.
I think a lot of people would feel like it should be removed.
Making this issue so it isn't forgotten about.

### LINKED ISSUE #16427: Windoor/firelock placement in Bagel Brig Entrance
dreamseeker_2017-11-02_18-09-00
The windoors are layered above the firelocks which make them extremely annoying to use.

### LINKED ISSUE #16453: Bagel: No Air Monitoring Computer in Telecoms Control
Looks like Bagel had the same oversight as Defficiency.

### LINKED ISSUE #16491: Bagelstation showers
(WEB REPORT BY: swagman55 REMOTE: 172.93.109.202:7777)
# Revision
bd483ea93c1801b5fd45f38f3d5b8c01fa4cc7ad

# Description
The shower curtains in the bathroom block the shower itself from being activated
# Steps to Reproduce

# What you Expected

# What Actually Happened

### LINKED ISSUE #16718: Baglestation Mining Station Airlock
(WEB REPORT BY: entya REMOTE: 172.93.109.202:7777)
# Revision
2183a76c8acde30338433b04228a1a6755fcd739

# Description
Make the airlocks cycle and remove the eemergency closet and o2 canister from the inside
# Steps to Reproduce
Try going outside
# What you Expected
To not get pinned and crushed 20 times
# What Actually Happened
I got pinned and crushed in the airlock 20 times

### LINKED ISSUE #16849: Prison cell 4 on bagel is bugged
If you set a timer for it, it will unlock the door and the locker almost immediately. The timer will still be ticking down during this process as normal.

### LINKED ISSUE #16872: The dorms on Bagelstation have no fire alarms.
Not that anyone uses them, but the dorms have 0 fire alarms. If the firelocks go up, you have to manually turn them all off.

### LINKED ISSUE #16873: Bagelstation causes AIs to crash randomly
#### Description of issue
Bagelstation causes AIs to have their Dreamseeker client crash at random. Various players have agreed that it's the map doing it. Could have sworn there was a formal issue for this already but apparently not.

#### Length of time in which bug has been known to occur
This has happened for months, maybe as long as the map has been in rotation.

#### Server revision

Game: a2e83a82c01a6fdb8a92c2e2e689db8726421f14 
Patches: 9e34bcea4d28270b966365362177f6890fc3ca45 

#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of not being yelled at by every contributor.

- [x] Issue could be reproduced at least once
- [x] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

### PR DESCRIPTION
**Pull request by CptWad**
hi

fixes #16873

fixed #16872

fixed #16718

The mining asteroid now has a proper gated airlock 
image

fixes #16491
example: 
image

closes #16453 as there IS one there

fixes #16427

fixed #16385

kick mine is rip
fixes #15920

closes #15107
duplicate

fixes #16849
fucking wow nice 

:cl:
 * bugfix: I am back from the dead! With bugfixes!
 * experimental: Removed the jukebox in the AI core to see if its the culprit of ais crashing
* rscadd: In order to make showers visible in shower curtains, added two new types of showers that allow visibility (click-ability) of shower heads while in the open position.

### CODE REVIEW

**Comment by ShiftyRail** [comment]
>fixes #16873 
D*g bless

**Comment by CptWad** [comment]
its more of an attempt
if this fails that means that jukeboxes are not the cause and I can put it back
then its more of the figuring out what is causing the crashing

**Comment by DamianX** [comment]
you...fixed #16873?

**Comment by CptWad** [comment]
Well nobody has told me WHY it happens only guesses that its the jukeboxes fault
so I axed the jukebox

**Comment by DamianX** [comment]
I see, let's see how it goes

File: maps/bagelstation.dmm [hunk 1]
```diff
@@ -1743,7 +1743,7 @@
 "aHA" = (/obj/machinery/atmospherics/pipe/simple/supply/hidden,/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"},/obj/structure/cable{d1 = 2; d2 = 8; icon_state = "2-8"},/turf/simulated/floor/plating,/area/maintenance/starboard)
 "aHB" = (/obj/machinery/atmospherics/pipe/simple/supply/hidden,/obj/structure/cable{d1 = 2; d2 = 8; icon_state = "2-8"},/obj/structure/cable{d1 = 1; d2 = 8; icon_state = "1-8"},/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"; pixel_y = 0},/turf/simulated/floor/plating,/area/maintenance/starboard)
 "aHC" = (/obj/structure/grille,/turf/simulated/floor/plating/airless,/area)
-"aHD" = (/obj/machinery/shower{dir = 4},/obj/structure/curtain/open/shower,/obj/item/device/radio/intercom{dir = 8; name = "Station Intercom (General)"; pixel_x = -28},/turf/simulated/floor{icon_state = "freezerfloor"},/area/crew_quarters/toilet)
+"aHD" = (/obj/structure/table,/obj/machinery/fishtank/bowl,/turf/simulated/floor/wood,/area/crew_quarters)
```
Comment (Kurfursten) [inline maps/bagelstation.dmm:1746]: why

File: maps/bagelstation.dmm [hunk 2]
```diff
@@ -6649,7 +6648,7 @@
 "cxS" = (/obj/machinery/light{dir = 4},/obj/machinery/atmospherics/unary/vent_scrubber/on{dir = 1},/obj/machinery/camera{c_tag = "Security Shuttle Dock"; dir = 8; name = "security shuttle dock camera"},/obj/item/device/radio/intercom{dir = 4; name = "Station Intercom (General)"; pixel_x = 31},/turf/simulated/floor{blocks_air = 0; icon_state = "dark"},/area/security/main)
 "cxT" = (/obj/machinery/light_switch{pixel_x = 0; pixel_y = -24},/turf/simulated/floor{tag = "icon-showroomfloor (NORTH)"; icon_state = "showroomfloor"; dir = 1},/area/security/toilet)
 "cxU" = (/obj/machinery/alarm{dir = 8; pixel_x = 24},/obj/machinery/washing_machine,/turf/simulated/floor{tag = "icon-showroomfloor (NORTH)"; icon_state = "showroomfloor"; dir = 1},/area/security/toilet)
-"cxV" = (/obj/structure/curtain/open/shower/security,/obj/machinery/shower{dir = 4},/turf/simulated/floor{tag = "icon-showroomfloor (NORTH)"; icon_state = "showroomfloor"; dir = 1},/area/security/toilet)
+"cxV" = (/obj/machinery/fishtank/wall,/turf/simulated/floor/plating,/area/crew_quarters/kitchen)
```
Comment (Kurfursten) [inline maps/bagelstation.dmm:6651]: are

File: maps/bagelstation.dmm [hunk 3]
```diff
@@ -7389,9 +7388,9 @@
...
+"cMh" = (/obj/structure/grille,/obj/structure/window/reinforced{dir = 1},/obj/structure/window/reinforced{dir = 4},/obj/structure/window/reinforced{dir = 8},/obj/structure/window/full/reinforced,/obj/structure/sign/botany,/obj/machinery/door/firedoor/border_only{dir = 1; name = "Firelock North"},/turf/simulated/floor/plating,/area/hydroponics)
 "cMi" = (/obj/machinery/atmospherics/pipe/simple/supply/hidden,/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden{dir = 4},/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"; pixel_y = 0},/turf/simulated/floor,/area/engineering/supermatter_room)
-"cMj" = (/obj/machinery/atmospherics/pipe/manifold/scrubbers/hidden{dir = 4},/obj/structure/closet/radiation,/turf/simulated/floor,/area/engineering/supermatter_room)
+"cMj" = (/obj/structure/closet/secure_closet/freezer/kitchen{req_access = null},/obj/item/weapon/fishtools/fish_egg_scoop,/obj/item/weapon/fishtools/fish_net,/obj/item/weapon/fishtools/fish_food,/obj/item/weapon/fishtools/fish_food,/obj/item/weapon/fishtools/fish_tank_brush,/turf/simulated/floor{icon_state = "showroomfloor"},/area/crew_quarters/kitchen)
```
Comment (Kurfursten) [inline maps/bagelstation.dmm:7393]: these

File: maps/bagelstation.dmm [hunk 4]
```diff
@@ -11041,11 +11040,11 @@
 "eeq" = (/turf/space,/area/research_outpost/hallway)
 "eer" = (/turf/unsimulated/floor/asteroid,/area/research_outpost/hallway)
 "ees" = (/turf/unsimulated/floor/airless{icon_state = "asteroidplating"},/area/research_outpost/hallway)
-"eet" = (/obj/machinery/recharger{pixel_y = -29},/obj/machinery/camera{c_tag = "Outpost Exit"; dir = 1; name = "exit camera"; network = list("MINE","RD","SS13")},/turf/unsimulated/floor/airless{icon_state = "asteroidplating"},/area/mine/explored)
+"eet" = (/obj/structure/sink/puddle,/obj/item/fish_eggs/goldfish,/obj/item/fish_eggs/goldfish,/turf/simulated/floor/grass,/area/hydroponics)
```
Comment (Kurfursten) [inline maps/bagelstation.dmm:11043]: not

File: maps/bagelstation.dmm [hunk 5]
```diff
@@ -11041,11 +11040,11 @@
...
-"eet" = (/obj/machinery/recharger{pixel_y = -29},/obj/machinery/camera{c_tag = "Outpost Exit"; dir = 1; name = "exit camera"; network = list("MINE","RD","SS13")},/turf/unsimulated/floor/airless{icon_state = "asteroidplating"},/area/mine/explored)
+"eet" = (/obj/structure/sink/puddle,/obj/item/fish_eggs/goldfish,/obj/item/fish_eggs/goldfish,/turf/simulated/floor/grass,/area/hydroponics)
 "eeu" = (/obj/machinery/recharger{pixel_y = -29},/turf/unsimulated/floor/asteroid,/area/mine/explored)
 "eev" = (/turf/simulated/wall/r_wall,/area/mine/production)
 "eew" = (/obj/structure/grille,/obj/structure/window/reinforced{dir = 8},/obj/structure/window/reinforced{dir = 4},/obj/structure/window/reinforced{dir = 1},/obj/structure/window/reinforced,/turf/simulated/floor/plating,/area/mine/production)
-"eex" = (/obj/machinery/door/airlock/external{name = "External Airlock"},/turf/simulated/floor/plating,/area/mine/production)
+"eex" = (/obj/structure/closet/secure_closet/personal/cabinet,/obj/item/clothing/glasses/welding,/obj/item/fish_eggs/salmon,/obj/item/fish_eggs/salmon,/turf/simulated/floor/plating,/area/maintenance/asmaint)
```
Comment (Kurfursten) [inline maps/bagelstation.dmm:11047]: in

File: maps/bagelstation.dmm [hunk 6]
```diff
@@ -12852,6 +12851,23 @@
...
+"eNw" = (/obj/machinery/door/airlock/external{id_tag = "MINE_airlock_interior"; name = "External Airlock"},/turf/simulated/floor/plating,/area/mine/production)
+"eNx" = (/obj/machinery/vending/mining,/obj/machinery/camera{c_tag = "Outpost Expedition Preperation"; name = "expedition camera"; network = list("MINE","RD","SS13")},/obj/machinery/airlock_sensor{frequency = 1331; id_tag = "MINE_int_airlock_sensor"; master_tag = "MINE_airlock_control"; pixel_x = 0; pixel_y = 22},/turf/simulated/floor,/area/mine/production)
+"eNy" = (/obj/structure/closet/crate,/obj/item/fish_eggs/shark,/obj/item/fish_eggs/shark,/turf/simulated/floor/wood,/area/research_outpost/hallway)
```
Comment (Kurfursten) [inline maps/bagelstation.dmm:12868]: the

File: maps/bagelstation.dmm [hunk 7]
```diff
@@ -12852,6 +12851,23 @@
...
+"eNz" = (/obj/structure/closet/crate,/obj/item/fish_eggs/electric_eel,/obj/item/fish_eggs/electric_eel,/obj/item/weapon/fishtools/fish_egg_scoop,/obj/item/weapon/fishtools/fish_food,/obj/item/weapon/fishtools/fish_food,/obj/item/weapon/fishtools/fish_net,/obj/item/weapon/fishtools/fish_tank_brush,/turf/simulated/floor/plating/vox,/area/vox_trading_post/vault)
```
Comment (Kurfursten) [inline maps/bagelstation.dmm:12869]: changelog?

**Pull request merged**